transactions/views.py

```diff
@@ -368,17 +371,20 @@ def validate_payment(request):
 
 
 @api_view(['GET'])
-@permission_classes((IsAuthenticated, ))
 def foreign_card_validation_response(request):
     """
     Endpoint for handling foreign card validation response. It uses the
     response to verify the payment and save the card token, if requested
     :param request: DRF request object
     :return: JSON response
     """
+    domain = os.environ.get('FRONT_END_URL')
+
     resp = json.loads(request.query_params['response'])
     verify_resp = TransactionServices.verify_payment(resp['txRef'])
-    user = request.user
+    # user = request.user
```

Comment:
@patrickf949 Let's remove the commented code

Reply:
Thanks @Victoradukwu, all done